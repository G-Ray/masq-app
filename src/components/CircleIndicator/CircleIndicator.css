.circlestatus {
  border-radius: 50%;
  width: 8px;
  height: 8px;
  /* border: solid 2px #ee6e7e; */
}
