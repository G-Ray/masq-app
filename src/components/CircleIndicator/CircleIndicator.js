import React from 'react'
import PropTypes from 'prop-types'

import './CircleIndicator.css'

export default function CircleIndicator (props) {
  return (
    <div className='circlestatus' style={{border: 'solid 2px' + props.color}} />
  )
}

CircleIndicator.propTypes = {
  color: PropTypes.string.isRequired
}
