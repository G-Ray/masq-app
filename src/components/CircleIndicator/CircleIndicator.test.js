import React from 'react'
import renderer from 'react-test-renderer'
import CircleIndicator from './CircleIndicator'

it('renders CircleIndicator with color blue', () => {
  const component = renderer.create(<CircleIndicator color='blue' />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

it('renders CircleIndicator with color red', () => {
  const component = renderer.create(<CircleIndicator color='red' />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
