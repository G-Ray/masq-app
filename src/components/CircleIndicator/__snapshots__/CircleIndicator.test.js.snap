// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CircleIndicator with color blue 1`] = `
<div
  className="CircleIndicator"
  style={
    Object {
      "border": "solid 2pxblue",
    }
  }
/>
`;

exports[`renders CircleIndicator with color red 1`] = `
<div
  className="CircleIndicator"
  style={
    Object {
      "border": "solid 2pxred",
    }
  }
/>
`;
